lint/backend.py

```diff
@@ -33,18 +34,33 @@ def lint_view(view, hit_time, callback):
     the corresponding section is linted as embedded code.
     """
     linters = get_linters(view)
-    lint_tasks = get_lint_tasks(linters, view)
+    lint_tasks = get_lint_tasks(linters, view, hit_time)
 
-    results = run_concurrently(
-        partial(execute_lint_task, *task, hit_time=hit_time)
-        for task in lint_tasks)
+    tasks_by_linter = defaultdict(list)
+    for linter, task in lint_tasks:
+        tasks_by_linter[linter].append(task)
 
-    all_errors = chain.from_iterable(results)
+    run_concurrently(
+        partial(run_tasks, tasks, next=partial(next, linter))
+        for linter, tasks in tasks_by_linter.items()
+    )
+
+
+def run_tasks(tasks, next):
+    results = run_concurrently(tasks)
```

Comment:
Just a note that this means we could have up to 25 threads linting concurrently. That's probably fine, but seems excessive.

Also, why do we need to group tasks by linter again?